lib/meta-box-partial-page.php

```diff
@@ -188,7 +217,6 @@ function gutenberg_meta_box_partial_page_admin_header( $hook_suffix, $current_sc
 		decimalPoint = '<?php echo addslashes( $wp_locale->number_format['decimal_point'] ); ?>',
 		isRtl = <?php echo (int) is_rtl(); ?>;
 	</script>
-	<meta name="viewport" content="width=device-width,initial-scale=1.0">
```

Comment:
Shouldn't be `viewport` kept?

Reply:
No this is not rendered at all, it's an AJAX request where we just render the `body`

Reply:
It makes perfect sense :)